lib/helpers/actions/click.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function(scope) {
+    return {
+        scope
```

Comment:
all actions and transforms helpers return descriptors with `type` property as a name of file.